packages/postcss-merge-longhand/types/lib/colornames.d.ts

```diff
@@ -1,14 +1,2 @@
-declare const _exports: {
-    add(value: string): Set<string>;
-    clear(): void;
-    delete(value: string): boolean;
-    forEach(callbackfn: (value: string, value2: string, set: Set<string>) => void, thisArg?: any): void;
-    has(value: string): boolean;
-    readonly size: number;
-    entries(): IterableIterator<[string, string]>;
-    keys(): IterableIterator<string>;
-    values(): IterableIterator<string>;
-    [Symbol.iterator](): IterableIterator<string>;
-    readonly [Symbol.toStringTag]: string;
-};
+declare const _exports: Set<string>;
```

Comment:
Looks like an improvement. Before it exported an object with all the methods of a Set instead.